app.py

```diff
@@ -447,23 +465,24 @@ async def view_submission(ack, body, logger, client):
     try:
         # formatting a message
         # todo: change to use json object
-        header_msg = f"*Slackblast*: "
+        header_msg = f"*Backblast!*: "
         title_msg = f"*" + title + "*"
 
-        date_msg = f"*DATE*: " + the_date
+        date_msg = f"*Date*: " + the_date
         ao_msg = f"*AO*: <#" + the_ao + ">"
-        q_msg = f"*Q*: <@" + the_q + ">"
         pax_msg = f"*PAX*: " + pax_formatted
         fngs_msg = f"*FNGs*: " + fngs
-        count_msg = f"*COUNT*: " + count
+        q_msg = f"*Q*: <@" + the_q + ">"
+        count_msg = f"*Total*: " + count
+        conditions_msg = f"*Conditions*: " + conditions
         moleskine_msg = moleskine
 
-        # Message the user via the app/bot name
+        # Message the channel as the user submitting
         if config('POST_TO_CHANNEL', cast=bool):
             body = make_body(date_msg, ao_msg, q_msg, pax_msg,
-                             fngs_msg, count_msg, moleskine_msg)
+                             fngs_msg, count_msg, conditions_msg, moleskine_msg)
             msg = header_msg + "\n" + title_msg + "\n" + body
-            await client.chat_postMessage(channel=chan, text=msg)
+            await client.chat_postMessage(channel=chan, text=msg, as_user=True)
```

Comment:
I was overjoyed when I saw this. Maybe we can post as the user and it is as easy as turning on a param! I was also thinking those people responding to SO about this must have overlooked this. 

But, I tried it, and sadly it doesn't work. The [docs](https://api.slack.com/methods/chat.postMessage#authorship) say it only works for classic Slack apps and I think we can only create a new Slack app. But if you found a way to get it to work, @kodesmith, then please correct me and tell me how you did it.

Reply:
I went through the changes before submitting this PR and missed this.  I meant to pull it back out.   I knew it wasn't working but I was trying to find a way to get the user's token.  Right now, the app is created with the bot token
`
slack_app = AsyncApp(
    token=config('SLACK_BOT_TOKEN'),
    signing_secret=config('SLACK_SIGNING_SECRET')
)
app_handler = AsyncSlackRequestHandler(slack_app)
`

But, my attempt was to try and get the user's token at some point and create a new "app_handler" to submit the backblast to the designated channel.  There's no great documentation and I hadn't taken the time to set up local debugging so I never made it that far.  Maybe someone else can run with that idea?

Either way, this won't make it into the final PR